### PR TITLE
Add W3ID for the public knowledge base of the CORE system

### PR DESCRIPTION
**Pull request by giachell**
The Collaborative Oriented RElation Extraction (CORE) system allows to create Knowledge Bases (KBs) by relying on relation extraction techniques.

The public knowledge base of CORE currently contains gene-diseases associations extracted from the scientific literature.

### CODE REVIEW

File: corekb/readme.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# The Collaborative Oriented RElation Extraction (CORE) system for building Knowledge Bases (KBs)
```
Comment (TallTed) [inline corekb/readme.md:1]:
```suggestion
# The Collaborative Oriented Relation Extraction (CORE) system for building Knowledge Bases (KBs) 
```

**Comment by giachell** [comment]
Thank you for the suggestions!
Now I should have fixed the typos.

**Comment by giachell** [comment]
Thank you for the suggestions!
Now I should have fixed the typos.

**Comment by dgarijo** [comment]
@giachell,
can you please add  yourself as a maintainer in the readme file?

**Pull request merged**